roles/docker/tasks/main.yml

```diff
@@ -3,18 +3,26 @@
   ansible.builtin.yum:
     name:
       - docker
-      - python2-pip
+      - python3-pip
       - docker-compose
     state: latest
     update_cache: true
   become: true
 
-- name: Upgrade pip to latest version that still supports Python 2.7
-  ansible.builtin.command: pip install pip==20.3.4
+# - name: Upgrade pip to latest version
+#   ansible.builtin.command: pip3 install -U pip
+#   become: true
+- name: Upgrade pip3 to latest version
+  ansible.builtin.pip:
+    name: pip
+    state: latest
+    executable: pip3
   become: true
```

Comment:
This will overwrite the version installed with the RPM using `ansible.builtin.yum`. Hence this cannot be "frozen" using the Pulp repos and could break other Python3 RPMs. Is this necessary? Or is the `python3-pip` RPM installed with `ansible.builtin.yum` good enough?

Reply:
There are a few other Python packages installed with pip3 and without version control:
1. Install docker-py (supported by python 3)
2. Install pyodbc with pip3
Can those be deployed in a python virtual environment for iRODS as opposed to in the default location affecting all Python code on the server?